js/store.js

```diff
@@ -134,19 +256,74 @@ const store = {
     const firstParty = this.getWebsite(origin);
     const thirdParty = this.getWebsite(target);
 
+/*
+  if not {},
+
+  firstParty is expected to match this format:
+  {
+    thirdPartyHostnames: [
+      "www.thirdpartydomain.com",
+      ...
+    ]
+  }
+  thirdParty is expected to match this format:
+  {
+    firstPartyHostnames: [
+      "www.firstpartydomain.com",
+      ...
+    ],
+    isIgnored: true, (optional)
+    whiteListedFirstParty: "www.google.com" (optional)
+  }
+  */
+
+    let whiteListOverride = false;
+    let onWhiteList = false;
+
     if (!('thirdPartyHostnames' in firstParty)) {
```

Comment:
This is Github trolling you... it collapsed the method name. I was confused too.